cmd/node_problem_detector.go

```diff
@@ -41,6 +42,36 @@ func validateCmdParams() {
 	}
 }
 
+func mustGetNodeName() string {
```

Comment:
Having thought that `must` prefix is a convention for function which may panic in Golang. Since `OrDie` is the convention for kubernetes, we should stick with it. :)